src/main/java/redis/clients/jedis/UnifiedJedis.java

```diff
@@ -3922,6 +3923,9 @@ public boolean bfExists(String key, String item) {
 
   @Override
   public List<Boolean> bfMExists(String key, String... items) {
+    if (items.length == 0) {
+      return Collections.emptyList();
+    }
```

Comment:
```suggestion
```
We don't do this in Jedis.

Reply:
Hi @sazzad16, varargs for items in function signature implicitly assumes 0 or more but redis expects 1 or more so we have to account for the 0 length (or we have to tell in documentation that 1 or more items are expected but i'd rather stick to regular understanding of function signature).

Reply:
@doumdoum `bfMExists` is not a simple straightforward word. The user who is calling bfMExists(), we assume that s/he has some idea about `BF.MEXISTS` command. We also take that documentation provided by Redis, e.g. https://oss.redis.com/redisbloom/Bloom_Commands/#bfmexists, is our prime documentation. Finally, we try to keep it as close to plain command as possible.

Reply:
@sazzad16 i can't agree with you: as a user of jedis i should not need to know redis protocol details, i should only rely on jedis API. Anyway i've remove the checks from the PR according to your proposal.